package.json

```diff
@@ -12,13 +12,14 @@
     "node": ">=18.0.0 <21.0.0"
   },
   "scripts": {
-    "merge-cover": "lcov-result-merger 'coverage/lcov_*.info' > coverage/general.info",
-    "coveralls": "cat coverage/general.info | coveralls",
+    "clean": "rimraf coverage",
```

Comment:
Do we need this cleaning for coverage?

Shouldn't we `run clean` recursively on all workspaces as we do in `ci`? I mean, if somebody runs `yarn clean` they expect it will clean the entire project, don't they?

Reply:
Thanks! Now 'clean' works recursively.